src/psl/psl-lower.c

```diff
@@ -114,37 +114,48 @@ static void psl_lower_state(lower_unit_t *lu, psl_fsm_t *fsm,
    if (state->initial && psl_fsm_repeating(fsm))
       psl_enter_state(state);
 
+   vcode_type_t vbool = vtype_bool();
+   vcode_reg_t vfalse = emit_const(vbool, 0);
+   vcode_reg_t vtrue = emit_const(vbool, 1);
+   vcode_var_t vanish = emit_var(vbool, vbool, ident_new("vanish"), VAR_TEMP);
```

Comment:
You don't need a variable here, the guard expressions can be OR-ed together. I pushed this separately in 8806798.

Reply:
I dropped my changes to `psl_lower_state` and rebased.